test/Microsoft.Health.Dicom.Web.Tests.E2E/Common/TestDicomWebServerFactory.cs

```diff
@@ -35,6 +35,34 @@ public static TestDicomWebServer GetTestDicomWebServer(Type startupType, bool en
         return new RemoteTestDicomWebServer(environmentUrl);
     }
 
+    private static Uri GetDicomServerUrl(DicomTestServerCategory testServerCategory)
+    {
+        var options = new TestEnvironmentOptions();
+        TestEnvironment.Variables.Bind(options);
+
+        if ((testServerCategory & DicomTestServerCategory.DataPartition) == DicomTestServerCategory.DataPartition)
+        {
+            return options.TestFeaturesEnabledEnvironmentUrl;
+        }
+
+        if ((testServerCategory & DicomTestServerCategory.Features) == DicomTestServerCategory.Features)
+        {
+            return options.TestFeaturesEnabledEnvironmentUrl;
+        }
+
+        if ((testServerCategory & DicomTestServerCategory.DualWrite) == DicomTestServerCategory.DualWrite)
+        {
+            return options.TestFeaturesEnabledEnvironmentUrl;
+        }
+
+        if (testServerCategory == DicomTestServerCategory.None)
+        {
+            return options.TestEnvironmentUrl;
```

Comment:
Could we not always just use `TestEnvironmentUrl` and be sure to set it appropriately based on what we want to test? For example

* E2E Tests
  * Set `TestEnvironmentUrl` to `dicom-ci`
  * `dotnet test --filter "Category!=bvt-fe&Category!=bvt-dp"`
* E2E Tests Features
  * Set `TestEnvironmentUrl` to `dicom-ci-features`
  * `dotnet test --filter "Category!=bvt-dp"`
* E2E Tests Data Partitions
  * Set `TestEnvironmentUrl` to `dicom-ci-dp`
  * `dotnet test --filter "Category=bvt-dp"`

Reply:
We dont have to test all the tests, so I think we should do 
E2E Tests
Set TestEnvironmentUrl to dicom-ci
dotnet test --filter "Category=bvt"